### PR TITLE
Try PPI_GEOTAG fix

### PR DESCRIPTION
**Pull request by gpotter2**
Not sure at all if it fixes https://github.com/secdev/scapy/issues/1091, but that bug was triggered on my computer anyway.

### CODE REVIEW

**Comment by guedou** [comment]
@p-l- can you have a look ?

File: scapy/contrib/ppi_geotag.uts [hunk 1]
```diff
@@ -30,11 +30,9 @@ assert GPSTime_Field("GPSTime", None).delta == 0.0
 
 = Define local_to_utc
 
+# BUG workaroung: on some Python 2 systems, summer time is ignored while converting time to UTC.
```
Comment (p-l-) [inline scapy/contrib/ppi_geotag.uts]: Why did you remove the Python 3 exception if the bug has been fixed?
